backend/matchmaking/consumers.py

```
@@ -94,6 +96,18 @@ async def match_player(self):
         while not match_found:
             player_in_queue: WaitingPlayer | None = await self.get_player_in_queue(player_to_match)
             matched_user: WaitingPlayer | None = await self.get_matched_user(player_to_match)
+            
+            if player_in_queue:
+                player_in_queue_username = await player_in_queue.get_username()
+                logger.debug(f"Player in queue: {player_in_queue_username}")
```

Pylint (3.3.7)

[warning] Use lazy % formatting in logging functions

(W1203)